api-docs/components/schemas/event.json

```diff
@@ -628,7 +628,20 @@
       },
       "errors": {
         "type": "array",
-        "items": {}
```

Comment:
This was the fix for the `schema not type of Schema` error.  It turns out you can't do an empty object of items, and thanks to the [literal only result on the internet](https://github.com/p1c2u/openapi-core/issues/256) for this I managed to fix it, and also realized I didn't do this in other places where I'd described `errors`.